main.go

```diff
@@ -31,7 +31,7 @@ type Function struct {
 
 	EnvironmentDelimiter string              `json:"environment_delimiter"`
 	Environment          []map[string]string `json:"environment"`
-
+	EnvironmentFile      string              `json:"env_vars_file"`
```

Comment:
Can you bring back the empty line plz? 
And a question about the `json` tag: this means the yaml will be `env_vars_file: ...` - right? 
Can you maybe name the struct field accordingly? Make it `EnvVarsFile` ? All others match the json tag.